server/routes/api.js

```diff
@@ -0,0 +1,87 @@
+const express = require("express");
+const router = express.Router();
+const moment = require("moment");
+const Expense = require("../models/Expense");
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+router.get("/expenses", function(req, res) {
+  if (req.query.d1) {
+    let d1 = moment(req.query.d1).format("LLLL");
+    let d2 = req.query.d2
+      ? moment(req.query.d2).format("LLLL")
```

Comment:
This is a duplicate code. You have the same code somewhere else, use one function to do that